tests/model_fields/tests.py

```diff
@@ -192,6 +193,10 @@ def test_formfield(self):
             with self.subTest(field=field):
                 self.assertIsInstance(field.formfield(), forms.ChoiceField)
 
+    def test_choices_from_enum(self):
+        # Check that Choices class was transparently resolved
```

Comment:
To add more explanation, comments should state the expected behavior and omit prefixes like "Check that" since all tests check things. This guideline is from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).